docs/output_fields.rst

```diff
@@ -435,3 +435,81 @@ Borehole samples (grondmonsters)
     fractie,10,float,0
     methode,10,string,ZEEFPROEF
 
+Bodemsites
+--------------------------------
+  .. csv-table:: Bodemsites
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemsite,1,string,https://www.dov.vlaanderen.be/data/bodemsite/2013-000180
+	naam,1,string,Meise_Neerpoorten
+	datum,1,date,nan
+	beschrijving,1,string,grasland
+	waarnemingsdatum,10,date,nan
+	invoerdatum,10,date,nan
+
+
+BodemLocaties
+--------------------------------
+  .. csv-table:: Bodemlocaties
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemlocatie,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/2011-000002
+	naam,1,string,STARC_4
+	type,1,string,profielput
+	datum,1,date,nan
+	doel,1,string,archeologische landschappelijke profielputten
+	x,1,float,206553.85
+	y,1,float,168891.11
+	z,1,float,44.00
+	erfgoed,1,boolean,true
+	bodemstreek,1,string,Zandleemstreek
+	waarnemingsdatum,10,date,nan
+	invoerdatum,10,date,nan
+	educatieve_waarde,10,string,ZEER
+
+
+Bodemmonsters
+--------------------------------
+  .. csv-table:: Bodemmonsters
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemmonster,1,string,https://www.dov.vlaanderen.be/data/bodemmonster/1964-264869
+	identificatie,1,string,KART_PROF_073E/12_H2_M1
+	datum_monstername,1,date,1964-11-12
+	tijdstip_monstername,10,time,NaN
+	type,1,string,ENK
+	monsternamedoor,1,string,Centrum voor Grondonderzoek (C.V.G.)
+	techniek,1,string,NaN
+	condities,1,string,Zie scan analoge profielbeschrijving
+	van,1,float,30
+	tot,1,float,45
```

Comment:
So you want:

Van -> diepte_van_cm
tot -> diepte_tot_cm
For the other fields
x and y-> It's available in WFS as gml:Point. How to parse this as x and y?
hoogte_referentie_mtaw -> This is the same location as their parent bodemlocatie (found in Bodemlocatiefiche).
So normally, to get x, y and reference-height, you can link it in code with the bodemlocatie-fiche. Is this enough or you prefer to have a x,y,z field in the WFS?

Reply:
Regarding x/y/z, it would be easiest if these were added in the WFS. It is similar to the Interpretaties: they too take their location from their parent object but have the fields available in the WFS.

Looking more closely I see that in pydov our Interpretatie types currently lack a "z" value. I think it would make sense to add it both here and there to always have the reference height next to the depth values.

I wouldn't call it `hoogte_referentie_mtaw` but use the same name as in the Bodemlocatie type: `mv_mtaw` or `start_bodemlocatie_mtaw` (in case of "maaiveld" or "aanvangspeil" respectively, see above)